tests/processes/LogPrinterThreadTest.py

```diff
@@ -2,14 +2,10 @@
 import unittest
 
 from coala_utils.ContextManagers import retrieve_stdout
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.processes.LogPrinterThread import LogPrinterThread
 
 
-class TestPrinter(LogPrinter):
-
-    def __init__(self):
-        LogPrinter.__init__(self, self)
+class TestPrinter():
```

Comment:
but if we remove the LogPrinterMixin we don't need to cover any line in this file ;)